lib/livebook/session.ex

```diff
@@ -1853,6 +1854,16 @@ defmodule Livebook.Session do
     {:noreply, handle_operation(state, operation)}
   end
 
+  def handle_info({:hub_changed, id}, %{data: %{notebook: %{hub_id: id}}} = state) do
```

Comment:
Is there a reason we don't do `{:hub_deleted, id}` instead?

Reply:
In the past, we decided to use `{:hub_changed, id}` for any crud operation, but we can revisit it since a lot of the implementation changed so far

Reply:
I see, semantically that would be nice, but it's up to you! :)